### DISCUSSION TITLE
Is there a way to set bionic reading in YAML frontmatter to true for creating PDF?

### DISCUSSION THREAD

**Discussion by Martin15135215** [quarto-dev/quarto-cli#7396] [Q&A]
### Description

Bionic Reading is bolding the first half of the word to guide the eye while reading.  While I am creating a paper with quarto, it would be nice to create a PDF for the draft version with words, where the first half is bolded. It was something my supervisor asked me.

**Comment by mcanouil** [comment]
Since you are using LaTeX, I suggest the following:
- https://tex.stackexchange.com/questions/644887/how-can-latex-form-an-environment-of-bionic-reading-style
- https://quarto.org/docs/journals/templates.html#latex-partials

**Reply by Martin15135215** [reply]
Thanks for the prompt answer. I look into this LaTeX way.

I thought there could be a quarto way.

**Reply by mcanouil** [reply]
LaTeX is LaTeX.
Since what you want seems to be a LaTeX thing, the "Quarto way" is to use partials/template to achieve what you want.

**Reply by Martin15135215** [reply]
I mean with the "Qaurto way" that I have the flexibility to transform my .qmd to pdf, html, docx... LaTeX is only specific to PDF.

**Reply by mcanouil** [reply]
Indeed, you could write a filter that does the transformation you need, see documentation on writing filter extensions.

**Reply by Martin15135215** [reply]
Thanks for the suggestion.
